src/broker/broker.c

```diff
@@ -847,6 +847,43 @@ receiver_thr_f (void *arg)
 	  continue;
 	}
 
+      if (strncmp(cas_req_header, "STATUS", 6) == 0)
+        {
+          int status = -1;
+          int pid, i, len;
+          unsigned int session_id;
+      
+          len = read_nbytes_from_client (clt_sock_fd, cas_req_header + SRV_CON_CLIENT_INFO_SIZE, 4);
+          if (len < 0)
+           {
+	         CLOSE_SOCKET (clt_sock_fd);
+	         continue;
+           }
+
+          memcpy ((char *) &pid, cas_req_header + 6, 4);
+          pid = ntohl (pid);
+          memcpy ((char *) &session_id, cas_req_header + 10, 4);
+          session_id = ntohl (session_id);
```

Comment:
Is session_id unique? Is there any possibility of duplication?

Reply:
session id is generated from cub_server and the id number is increased for each connection, so the id is unique.